plugins/processors/dcos_metadata/dcos_metadata.go

```diff
@@ -23,6 +24,9 @@ import (
 	"github.com/mesos/mesos-go/api/v1/lib/httpcli/httpagent"
 )
 
+const dcosMetricsPrefix = "DCOS_METRICS_"
+dcosMetricsLabelsWhitelist := [...]string{"DCOS_SERVICE_NAME"}
```

Comment:
Would it make sense to retrieve these values from the plugin's config, rather than hardcoding them?